packages/gotop/build.sh

```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/xxxserxxx/gotop
+TERMUX_PKG_DESCRIPTION="A terminal based graphical activity monitor inspired by gtop and vtop"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@medzikuser"
+_MAJOR_VERSION=4.1.2
+# Use the ~ deb versioning construct in the future:
+TERMUX_PKG_VERSION=1:${_MAJOR_VERSION}
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SRCURL=https://github.com/xxxserxxx/gotop/archive/refs/tags/v${_MAJOR_VERSION}.tar.gz
+TERMUX_PKG_SHA256=81518fecfdab4f4c25a4713e24d9c033ba8311bbd3e2c0435ba76349028356da
+TERMUX_PKG_NO_STATICSPLIT=true
```

Comment:
Does package provides static libraries which are for specific reasons shouldn't be splitted? If not, remove no-staticsplit var.